index.js

```diff
@@ -1,32 +1,39 @@
+'use strict';
 require('document-register-element');
 var copy = require('copy-to-clipboard');
 var CopyButtonPrototype = Object.create(HTMLButtonElement.prototype);
 
-function handleCopyClick(e) {
-  var toCopy, toCopyEl;
-  if (toCopy = this.getAttribute('target-text')) {
-    return copy(toCopy);
+function handleCopyClick(event) {
+  if (event.defaultPrevented) return;
```

Comment:
This one is breaking, too. Now mimics other controls. Default behavior may be prevented on capturing.